pkg/distributor/push_test.go

```diff
@@ -846,6 +845,114 @@ func TestHandler_ErrorTranslation(t *testing.T) {
 		})
 	}
 }
+func TestHandler_HandleRetryAfterHeader(t *testing.T) {
```

Comment:
not sure why the linter isn't complaining
```suggestion

func TestHandler_HandleRetryAfterHeader(t *testing.T) {
```

Reply:
what's the change here?

Reply:
An empty line between lines 847 and 848 :)